gha-webhook-listener.py

```diff
@@ -22,14 +23,19 @@
 from __future__ import print_function
 
 import argparse
+from dateutil import parser as dateparser
```

Comment:
this doesn't actually seem to be used? could you remove it, and remove the dependency from `setup.py`

Reply:
Indeed, leftover from a previous iteration